Wikipedia/Code/SectionEditorViewController.swift

```diff
@@ -250,6 +255,14 @@ extension SectionEditorViewController: SectionEditorNavigationItemControllerDele
     func sectionEditorNavigationItemController(_ sectionEditorNavigationItemController: SectionEditorNavigationItemController, didTapRedoButton redoButton: UIBarButtonItem) {
         messagingController.redo()
     }
+    
+    func sectionEditorNavigationItemController(_ sectionEditorNavigationItemController: SectionEditorNavigationItemController, didTapReadingThemesControlsButton readingThemesControlsButton: UIBarButtonItem) {
+        
+        messagingController.blur()
```

Comment:
`webView.resignFirstResponder()` may work here w/o need for new new js `blur()`

Reply:
👍 done, and it fixed that bug too, thanks!

Reply:
🕺🎉